src/Bicep.Core/Configuration/ExperimentalFeaturesEnabled.cs

```diff
@@ -8,14 +8,15 @@
 namespace Bicep.Core.Configuration;
 
 public record ExperimentalFeaturesEnabled(
-    bool? SymbolicNameCodegen,
-    bool? Extensibility,
-    bool? ResourceTypedParamsAndOutputs,
-    bool? SourceMapping,
-    bool? UserDefinedTypes,
-    bool? UserDefinedFunctions)
+    bool SymbolicNameCodegen,
+    bool Extensibility,
+    bool ResourceTypedParamsAndOutputs,
+    bool SourceMapping,
+    bool UserDefinedTypes,
+    bool UserDefinedFunctions,
+    bool PrettyPrinting)
```

Comment:
>PrettyPrinting [](http://example.com/codeflow?start=9&length=14)

It will still default to `false` if the experimental feature is not specified in the config file?

Reply:
Yes, all Boolean configs default to `false`.